spec/models/user_spec.rb

```diff
@@ -271,4 +271,75 @@
       expect(duplicates).to eq(3)
     end
   end
+
+  describe "password_resets_limit_reached?" do
+    context "user with no password reset requests" do
```

Comment:
`context` should start with a `when` or `with[out]`... so
```suggestion
    context "with 0 resets requested" do
```